### PR TITLE
error in sentax

### PR DESCRIPTION
**Pull request by ShehabAhmedAdel**

### CODE REVIEW

File: HAL/Keypad/Keypad.c [hunk 1]
```diff
@@ -41,21 +41,21 @@ UINT8 Keypad_GetKey(void)
...
 	{
-		GPIO_WRITE_PIN_VALUE(PORT_B ,ROWCOUNT , LOW);
+		GPIO_WRITE_PIN_VALUE(KEYPAD_PORT ,ROWCOUNT , LOW);
 		for (COLCOUNT=COL_INT ;COLCOUNT<=COL_END ;COLCOUNT++)
 		{
-			if(GPIO_READ_PIN_VALUE(PORT_B,COLCOUNT)==LOW)
+			if(GPIO_READ_PIN_VALUE(KEYPAD_PORT,COLCOUNT)==LOW)
 			{
-				RESULT=keypad_array[ROWCOUNT][COLCOUNT-COL_INT];
+				RESULT=keypad_array[ROWCOUNT-ROW_INT][COLCOUNT-COL_INT];
```
Comment (IslamMohamed8) [inline HAL/Keypad/Keypad.c:49]: why ?

File: HAL/Keypad/Keypad.c [hunk 2]
```diff
@@ -36,8 +36,8 @@ void Keypad_Init(void){
 
 UINT8 Keypad_GetKey(void)
 {
-  UINT8 ROWCOUNT;
-	UINT8 COLCOUNT;
+  UINT8 ROWCOUNT=0;
```
Comment (IslamMohamed8) [inline HAL/Keypad/Keypad.c:39]: Initialized in the loop

File: HAL/Keypad/Keypad.c (hunk 1 above)
Comment (ShehabAhmedAdel) [reply HAL/Keypad/Keypad.c:49]: Spare

File: HAL/Keypad/Keypad.c (hunk 2 above)
Comment (ShehabAhmedAdel) [reply HAL/Keypad/Keypad.c:39]: sorry, I have overlap in my mind.

File: HAL/Keypad/Keypad.c (hunk 1 above)
Comment (EsamMohamedSrour) [reply HAL/Keypad/Keypad.c:49]: already row_int  =  zerooo

Comment (ShehabAhmedAdel) [reply HAL/Keypad/Keypad.c:49]: ok,as you like

**Pull request closed without merging**